EndlessClient/UIControls/ScrollBar.cs

```diff
@@ -88,7 +89,7 @@ public void ScrollToTop()
         {
             ScrollOffset = 0;
             var pixelsPerLine = (float)(scrollArea.Height - _scrollButton.DrawArea.Height * 2) / (_totalHeight - LinesToRender);
-            _scrollButton.DrawPosition = new Vector2(_scrollButton.DrawArea.X, _scrollButton.DrawArea.Height + pixelsPerLine * ScrollOffset);
+            _scrollButton.DrawPosition = new Vector2(_scrollButton.DrawArea.X, 1 + _scrollButton.DrawArea.Height + pixelsPerLine * ScrollOffset);
```

Comment:
Since `ScrollOffset` is zero, the ` + pixelsPerLine * ScrollOffset` could be removed (it will always be zero). Oversight on my part when I first wrote it